packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -256,13 +252,7 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
         dataStore._root.set("my blob", blobHandle);
         assert.strictEqual(bufferToString(await (dataStore._root.get("my blob")).get(), "utf-8"), text);
 
-        const attachP = container.attach(provider.driver.createCreateNewRequest(provider.documentId));
-        if (provider.driver.type !== "odsp") {
```

Comment:
this should stay

Reply:
oh so this is passing the error to itExpects.